### PR TITLE
Support v3 of the Lynx tool

### PR DESCRIPTION
**Pull request by paulbrown1982**
## What are you doing in this PR?
Change to also generate acquisitionData from 5 utm_* parameters iff they are all present. This is to support v3 of the lynx tool.

[**Trello Card**](https://trello.com/c/wxnDQWVD)

## Why are you doing this?
Improving the lives of the Marketing and D&I teams in evaluating the effectiveness of marketing across our estate in BigQuery. See https://docs.google.com/document/d/1e3U3LK8OOa-mWrSvFwhyHFG7jL5r4K6NFKSmI9VSB8w/edit

## How to test
1. Generate a CODE environment URL using the new v2 link builder: https://reader-revenue-lynx.s3.eu-west-1.amazonaws.com/v2.html. Visit URL in an Incognito window.
-  Check that the acquisitionData session storage is as expected
2. Generate a CODE environment URL using the new v3 link builder: https://reader-revenue-lynx.s3.eu-west-1.amazonaws.com/v3.html. Visit URL in an Incognito window.
-  Check that the acquisitionData session storage is identical to the two times before (except for the recording of what the query string is).
3. Override the SOURCE value in the `&acquisitionData=...` parameter and confirm it takes precedence over the `utm_` data.
-  Check that the acquisitionData session storage contains an updated source and that the rest are identical to the two times before (except for the recording of what the query string is).
4. Remove `acquisitionData=...` parameter from the URL, keeping the `utm_*` ones. Visit URL in a new Incognito window.
-  Check that the acquisitionData session storage is identical to the first two times before (except for the recording of what the query string is).

## How can we measure success?

1. v1 and v2 tools can be turned off.
2. v3 tool can be configured not to add `acquisitionData=...` to the URL

## Have we considered potential risks?
Marketing know to fall back to using v2.html

## Accessibility test checklist
N/A

## Screenshots
N/A

### CODE REVIEW

File: support-frontend/assets/helpers/tracking/acquisitions.ts [hunk 1]
```diff
@@ -269,6 +269,37 @@ function getReferrerAcquisitionDataFromSessionStorage():
 		: null;
 }
 
+function getAcquisitionDataFromUtmParams():
+	| Record<string, unknown>
```
Comment (rupertbates) [inline support-frontend/assets/helpers/tracking/acquisitions.ts:273]: Can we type this to anything more specific than `unknown`?

**Review by rupertbates — APPROVED** [review]
Just one question, looks good though.

**Review by GHaberis — APPROVED** [review]
Looks good

File: support-frontend/assets/helpers/tracking/acquisitions.ts (hunk 1 above)
Comment (paulbrown1982) [reply support-frontend/assets/helpers/tracking/acquisitions.ts:273]: Yup, can do `	| Record<string, string | AcquisitionABTest | AcquisitionQueryParameters>`. Patch incoming.

**Pull request merged**